Storage/src/Bucket.php

```diff
@@ -267,6 +267,18 @@ public function exists(array $options = [])
      *           Acceptable values include, `"authenticatedRead"`,
      *           `"bucketOwnerFullControl"`, `"bucketOwnerRead"`, `"private"`,
      *           `"projectPrivate"`, and `"publicRead"`.
+     *     @type array $retention Object retention configuration.
+     *     @type string $retention.retainUntilTime The earliest time in RFC3339
+     *           UTC "Zulu" format that the object can be deleted or replaced.
+     *           This is the retention configuration set for this object.
+     *     @type string $retention.mode The mode of the retention configuration,
+     *           which can be either `"Unlocked"` or `"Locked"`.
+     *     @type string $retentionExpirationTime The earliest time in
```

Comment:
Looks similar to `retainUntilTime`. Could you please explain the use case of this parameter?

Reply:
Added a clarification to resolve this.

`retainUntilTime` is for the object.

`retentionExpirationTime` is the effective time of retention, basically max of bucket's retention time and object's retention time.

I changed the language `and` ==> `as well as` to make it clearer.